client/src/features/Event/Detail/containers/EventDetailContainer.tsx

```
@@ -4,7 +4,50 @@ import { Flex } from '@/shared/components/Flex';
 
 export const EventDetailContainer = ({ children }: PropsWithChildren) => {
   return (
-    <Flex dir="column" gap="24px" margin="60px 0 0 0" padding="40px 20px 0 20px">
+    <Flex
+      as="main"
+      id="event-main"
+      role="main"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)